src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/IndexViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class IndexViewModel
+    {
+        public string AccountHashedId { get; set; }
+
+        public IEnumerable<ApprenticeshipDetailsViewModel> Apprenticeships { get; set; }
+        public string SortedByHeaderClassName { get; set; }
+        public const string HeaderClassName = "das-table__sort";
+    
+        public void SortedByHeader()
+        {
+            SortedByHeaderClassName += HeaderClassName;
+            if (FilterModel.ReverseSort)
+            {
+                SortedByHeaderClassName += " das-table__sort--desc";
+            }
+            else
+            {
+                SortedByHeaderClassName += " das-table__sort--asc";
+            }
+        }
+
+        public ApprenticesFilterModel FilterModel { get; set; }
```

Comment:
These should be with the other properties of the class unless there is a reason for them alone being after the method?